dali/python/nvidia/dali/external_source.py

```diff
@@ -321,13 +376,30 @@ class ExternalSource():
     of separate Tensors, there will be an additional copy made internally, consuming both memory
     and bandwidth.
 
+    Automatically set to ``True`` when ``parallel=True``
+
 `batch` : bool, optional
     If set to ``True`` or ``None``, the ``source`` is expected to produce an entire batch at once.
     If set to ``False``, the ``source`` is called per-sample.
+
+`parallel` : bool, optional, default = False
+    If set to ``True`` the corresponding pipeline will run pool of Python workers to run the
+    callback in parallel. You can specify the number of workers by passing ``py_workers_num``
+    into pipeline's constructor.
+
+    When parallel=True ``source`` must return NumPy/MXNet/PyTorch CPU array, TensorCPU, or
```

Comment:
```suggestion
    When parallel=True ``source`` must return NumPy/MXNet/PyTorch CPU array, TensorCPU (or anyt other `array_interface` compatible data), or
```

Reply:
We don't mention it for regular ExternalSource, I'd say let's try to not go over 3k lines and fill the gaps in a follow up.

Reply:
Sure